client/client.go

```diff
@@ -315,6 +315,9 @@ func NewClient(cfg *config.Config, consulCatalog consul.CatalogAPI, consulServic
 	// Initialize the server manager
 	c.servers = servers.New(c.logger, c.shutdownCh, c)
 
+	// Start server manager rebalancing go routine
+	go c.servers.Start()
```

Comment:
Curious how this call was dropped?  Was it a regression?

Reply:
I don't think it was ever there so for long running clients only failed rpcs or discovering new servers would cause reshuffling.